src/antigravity/memory.py

```
@@ -0,0 +1,259 @@
+
+        Args:
+            session_id: Unique session identifier
+
+        Returns:
+            Path to saved file
+        """
+        filepath = self.storage_path / f"{session_id}.json"
+
+        data = {
+            "session_id": session_id,
+            "saved_at": datetime.now().isoformat(),
```

Ruff (0.14.8)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)